Harvest.Web/Views/Error/Index.cshtml

```diff
@@ -130,6 +130,11 @@
             }
         }; 
     </script>
+
+    <script type="text/javascript">
+        //Set copywrite year
+        document.getElementById("copywrite-year").innerHTML = new Date().getFullYear();
```

Comment:
@jSylvestre wrong spelling of copyright

Reply:
Yeah, but that says to write the year :)